Open Judge System/OJS.Common/Models/ProblemGroupType.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace OJS.Common.Models
+{
+    using OJS.Common.Attributes;
+
+    using Resource = Resources.Enums.EnumTranslations;
+
+    public enum ProblemGroupType
+    {
+        [LocalizedDescription("None", typeof(Resource))]
+        None = 0,
```

Comment:
Consider naming the default ProblemGroupType to `IncludedInHomework`. Or do you think it would be detrimental for some reason?

Reply:
This type does not reside in the database and is used only for displaying and checking purposes and indeed is meant to be treated as 'None' (as the default value of Enum is always 0)